tests/fixtures/markdown/code.md

```diff
@@ -0,0 +1,8 @@
+Here is some Python:
+
+```python
```

Comment:
As it's a text fixture I really don't see the point of changing this. If I touch it, maybe I can add a new block with py so the test ensuer both?